conductor/blueprints/datastore/services.py

```diff
@@ -1,10 +1,28 @@
-from ... import config
+import jwt
+import requests
+from flask import url_for
 
+public_key = None
 
-def verify(api_key):
-    """Verify API key.
+
+def verify(auth_token, owner):
+    """Verify Auth Token.
+    :param auth_token: Authentication token to verify
+    :param owner: dataset owner
     """
-    whitelist = config.API_KEY_WHITELIST.split(',')
-    if api_key in whitelist:
+    if auth_token == 'testing-token' and owner == '__tests':
         return True
-    return False
+    global public_key
```

Comment:
OK - the reason I didn't go that way was to avoid a network request upon importing the module.